amethyst_utils/src/ortho_camera.rs

```diff
@@ -5,39 +5,96 @@ use amethyst_core::specs::{
 use amethyst_core::Axis2;
 use amethyst_renderer::{Camera, ScreenDimensions};
 
+/// The coordinates that `CameraOrtho` will keep visible in the window
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct CameraOrthoWorldCoordinates {
+    /// Left x coordinate
+    pub left: f32,
+    /// Right x coordinate
+    pub right: f32,
+    /// Bottom y coordinate
+    pub bottom: f32,
+    /// Top y coordinate
+    pub top: f32,
+}
+
+impl CameraOrthoWorldCoordinates {
+    /// Creates coordinates with (0,0) at the bottom left, and (1,1) at the top right
+    pub fn normalized() -> CameraOrthoWorldCoordinates {
+        CameraOrthoWorldCoordinates {
+            left: 0.0,
+            right: 1.0,
+            bottom: 0.0,
+            top: 1.0,
+        }
+    }
+
+    /// width / height of the desired camera coordinates
+    pub fn aspect_ratio(&self) -> f32 {
+        self.width() / self.height()
+    }
+
+    /// size of the x-axis
+    pub fn width(&self) -> f32 {
+        self.right - self.left
+    }
+
+    /// size of the y-axis
```

Comment:
```rust
/// Returns the size of the X axis.
```

also, we should document that `top` should be more positive than `bottom` in the struct definition, and perhaps add a `new(..)` function that validates that